library/src/main/java/com/novoda/downloadmanager/DownloadManagerBuilder.java

```diff
@@ -5,6 +5,11 @@
 
 import com.novoda.downloadmanager.lib.DownloadManager;
 
+/**
+ * A client can specify whether the downloads are allowed to proceed by implementing
+ * {@link com.novoda.downloadmanager.lib.DownloadClientReadyChecker} on your Activity class
```

Comment:
maybe, i just think we should be careful to not clutter the readme (even the parralel thing prob shouldn't be in the readme)